### PR TITLE
Updating api to allow blank strs for query params, updating response to include 'after', updating tests

### PR DESCRIPTION
**Pull request by bsieber-mozilla**
#140 ref

### CODE REVIEW

File: ctms/app.py [hunk 1]
```diff
@@ -255,12 +255,20 @@ def get_bulk_contacts_by_timestamp(
...
         end=end_time,
+        after=url_safe_encoded_str,
         limit=limit,
         items=results,
         next=next_url,
     )
 
 
+def updates_helper(value, default):
+    blank_vals = ["", " ", None]
```
Comment (jwhitlock) [inline ctms/app.py:266]: It seems to me that `blank_vals = ["", None]` should be the list. Was a space (the string `" "`) needed to make this work? If it was, then should any whitespace string be permitted?

**Review by jwhitlock — APPROVED** [review]
Thanks @bsieber-mozilla, this worked well in my local testing.

File: ctms/app.py (hunk 1 above)
Comment (bsieber-mozilla) [reply ctms/app.py:266]: I can remove this! I wasn't sure if there were any limitations on the cinchy side; a single whitespace would be allowed when this is removed, but would likely break the system. So this was a safety net.

File: ctms/app.py [hunk 2]
```diff
@@ -255,12 +255,20 @@ def get_bulk_contacts_by_timestamp(
     return CTMSBulkResponse(
         start=start_time,
         end=end_time,
+        after=url_safe_encoded_str,
```
[the remark this answers is not in the document]
Comment (bsieber-mozilla) [reply ctms/app.py:258]: Sure, I didn't name this variable in this PR, just reused it, but I will do the needful!

[the remark this answers is not in the document]
Comment (bsieber-mozilla) [reply ctms/app.py]: `after_encoded` is the adjusted name

File: ctms/app.py (hunk 1 above)
Comment (bsieber-mozilla) [reply ctms/app.py]: Removed the whitespace to just have blank_vals be `["", None]`

**Review by jwhitlock — APPROVED** [review]
Thanks for the nit fixes!

**Pull request merged**